runtime/gc_include/ObjectAllocationAPI.hpp

```diff
@@ -173,6 +174,9 @@ class MM_ObjectAllocationAPI
 						/* Do not zero the TLH if it is already zero's */
 						initializeSlots = initializeSlots && _initializeSlotsOnTLHAllocate;
 #endif /* J9VM_GC_BATCH_CLEAR_TLH */
+#if defined(J9VM_GC_REALTIME)
+						initializeSlots = initializeSlots || (MM_ObjectAccessBarrierAPI::isSATBBarrierEnabled(currentThread));
+#endif /* J9VM_GC_REALTIME */
```

Comment:
We'll accept this, but likely this is likely overkill (seems like we will explicitly clear am object/array even if TLH was batch cleared). Please test later with relaxed approach (probably you just need to swap the two assignment lines with initializeSlots).

Reply:
I think the platform of interest to be tested is AIX what this the only one that has batch TLH clearing (@dmitripivkine will correct me if I'm wrong)

Reply:
I think all PPC platforms support Batch Clear TLH: AIX, Linux PPC BE and Linux PPC LE